tests/Fixer/LanguageConstruct/ClassKeywordRemoveFixerTest.php

```diff
@@ -270,6 +270,9 @@ public function c() {
                 echo ClassB::class;
                 echo C::class;
                 ',
+        ];
+
```

Comment:
This [got](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/5068) broken in 2020.